src/app/app.module.ts

```diff
@@ -68,6 +69,7 @@ export function lottiePlayerFactory() {
     Device,
     httpInterceptorProviders,
     { provide: ErrorHandler, useClass: ErrorHandlerService },
+    PLHAssetPipe,
```

Comment:
nit(blocking)
I'm confused why we are declaring the pipe as a provider in the main module, is this an import error or do we need for some reason (if so maybe a comment would help)

Reply:
I wanted to use the pipe in the component files, e.g. [here](https://github.com/IDEMSInternational/parenting-app-ui/pull/2028/files#diff-0511da63615ec05d036268319070a77820fd7aa64b314c90113c8d23fb19718dR115) in the audio component file. Using the pipe just in the html would lead to complicated logic (we want to check if the author has provided an icon override path and only apply the asset pipe to the icon path property if they have, but we also want to use a different icon path depending on whether the audio is playing, in order to display the play/pause icon).

Previously, that file imported the `TemplateAssetService` itself, which is used by the `plhAsset` pipe, but importing the pipe seemed like a neater way to do it to avoid code duplication. 

In order to inject the pipe, it needs to be declared in a relevant module. I've now moved it to the [template module file](https://github.com/IDEMSInternational/parenting-app-ui/pull/2028/files#diff-2ad3d485b13847f8402e2242cd65f73e57242ce3f8c1c0672e62a2718a0f63caR48), which seems like a better place than the main module. 

Alternatively, we could avoid injecting the `plhAsset` pipe directly, which would mean that it wouldn't need to be declared as a provider. But in that case, we would still need to import and inject the template asset service into our component in order to pass it to the constructor of the pipe, something like this:
```
constructor(private templateAssetService: TemplateAssetService) {
  super();
  this.plhAssetPipe = new PLHAssetPipe(this.templateAssetService)
}
```

I'm not clear on what the best option would be, but I can see how we might want to potentially inject other pipes into components in the future, so having them [declared as providers in the template module](https://github.com/IDEMSInternational/parenting-app-ui/pull/2028/files#diff-2ad3d485b13847f8402e2242cd65f73e57242ce3f8c1c0672e62a2718a0f63caR48) doesn't seem unreasonable.